build.gradle

```diff
@@ -14,6 +14,22 @@ apply plugin: 'nebula-aggregate-javadocs'
 apply plugin: 'org.ajoberstar.git-publish'
 apply plugin: 'net.researchgate.release'
 
+task (type: Jar, overwrite: true) {
```

Comment:
This line's not working. I think it's creating a new unnamed task, rather than overriding the default task.

Reply:
See also comment above, it might show up as being on an outdated commit

Reply:
You can check if it is working using `gradlew pTML` to publish to maven local, then unzipping one of the jar files from `~/.m2/repository/org/concordion/cubano-core/0.2.2/cubano-core-0.2.2.jar` and checking the `META-INF/MANIFEST.MF` file.